ibis/pandas/execution/tests/test_temporal.py

```diff
@@ -163,12 +170,19 @@ def test_times_ops_py2(t, df):
 @pytest.mark.parametrize(
     ('op', 'expected'),
     [
-        (lambda x, y: x + y, lambda x, y: x.values * 2),
-        (lambda x, y: x * 2, lambda x, y: x.values * 2),
-        pytest.mark.xfail(
-            (lambda x, y: x // 2, lambda x, y: x.values // 2),
-            raises=TypeError, reason='Not yet tested'
-        )
+        param(lambda x, y: x + y, lambda x, y: x.values * 2, id='add'),
+        param(lambda x, y: x * 2, lambda x, y: x.values * 2, id='mul'),
+        param(
+            lambda x, y: x // 2, lambda x, y: x.values // 2, id='floordiv',
+            marks=pytest.mark.xfail(
+                parse_version(pd.__version__) < parse_version('0.23.0'),
+                raises=TypeError,
+                reason=(
```

Comment:
do we have a issue on pandas side for this? maybe add a refernce?

Reply:
I think this is just a case of "added in a later version", which is fine. In theory we could implement support for it here by doing the division ourselves, but that should happen in a different issue/PR.

Reply:
Created an issue for it: https://github.com/ibis-project/ibis/issues/1550